client-v2/src/redux/modules/pageViewData/actions.ts

```diff
@@ -61,12 +61,12 @@ const convertToStoriesData = (
 
 const addArticleId = (
   ophanData: PageViewDataFromOphan,
-  articles: ArticlePathAndId[]
+  articles: DerivedArticle[]
 ): string => {
   // the path from ophan has a slash at the front, removing below
   const ophanPathClean = ophanData.path.substr(1);
-  const matchingArticle = articles.find(a => a.articlePath === ophanPathClean)!;
-  return matchingArticle.articleId;
+  const matchingArticle = articles.find(a => a.urlPath === ophanPathClean)!;
+  return matchingArticle.id;
```

Comment:
This appears to be returning as `undefined` and leading to no graph data displaying

Reply:
Gotcha, it's likely a result of me removing the intermediate type `ArticleWithPath` or similar.